boxsdk/config.py

```diff
@@ -9,3 +9,9 @@ class API(object):
     UPLOAD_URL = 'https://upload.box.com/api/2.0'
     OAUTH2_API_URL = 'https://api.box.com/oauth2'  # <https://developers.box.com/docs/#oauth-2>
     OAUTH2_AUTHORIZE_URL = 'https://app.box.com/api/oauth2/authorize'  # <https://developers.box.com/docs/#oauth-2-authorize>
+
+
+class Version(object):
```

Comment:
I'd like `USER_AGENT_STRING` to be user configurable. However, I agree that version isn't a config.

version.py should work